algorithms/src/Kokkos_Sort.hpp

```diff
@@ -66,6 +65,13 @@
 
 #endif
 
+#if defined(KOKKOS_ENABLE_ONEDPL)
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/algorithm>
+#endif
+
+#include <algorithm>
```

Comment:
Why did you move this include?

Reply:
To minimize the chances that we get clashing `TBB` and `oneTBB`. With the preprocessor macros, this is not necessary anymore, though.

Reply:
Then please undo this change.